GVFS/GVFS.Installer/Setup.iss

```diff
@@ -278,6 +283,18 @@ begin
     end;
 end;
 
+procedure WriteGitStatusCacheAvailableFile();

```

Comment:
In general, I think this pattern is currently too complex, and unfortunately not something we can add functional tests for because of the need to reboot to trigger the behavior. I would definitely like to see more manual testing to ensure that it is working, and it would be nice to add functional tests that at least cover all of the logic. E.g. query the last reboot time, and then (following my upgrades.dat pattern), set up the last upgrade time to be before the actual reboot time vs after the actual reboot time, and confirm that the GitStatusCache responds correctly.